core/iwasm/include/wasm_export.h

```diff
@@ -2258,7 +2259,37 @@ WASM_RUNTIME_API_EXTERN wasm_shared_heap_t
 wasm_runtime_create_shared_heap(SharedHeapInitArgs *init_args);
 
 /**
- * Attach a shared heap to a module instance
+ * This function links two shared heaps, `head` and `body`, where `head` is a
+ * shared heap and `body` can be shared heap chain head, into a single chain.
```

Comment:
a little confused about "*where `head` is a shared heap and `body` can be shared heap chain head*"